### ISSUE TITLE
Minidump path: as default

### ISSUE THREAD

**Issue by nnian** [qwertycoin-org/qwertycoin#127]
Hi @speqtr,

when i start qwertycoind, or in this case the `simplewallet` :
I can't use the daemon or simplewallet, only via RPC commands.

```
foo@bar:/opt/QWC# ./simplewallet    
Minidump path:
```

**Comment by speqtr** [comment]
@nnian does it crash instantly?

**Comment by nnian** [comment]
It does not crash, it works in the background but i can't use the daemon or simplewallet.
Other pool owners having the same issue.
On a fresh installed Ubuntu 18.04 VM this error occurs too.

**Comment by speqtr** [comment]
Got it. I'll check it tomorrow on a clean Ubuntu 18.04 VM.

Just one more thing... which scenario should I use to confirm that it works as you expect it to?

**Comment by exploshot** [comment]
Proceed as in readme. Thats the most common way.

**Comment by speqtr** [comment]
OK.

**Comment by nnian** [comment]
I only start the ./simplewallet that's it.

```
root@poo:/opt/qwertycoin# ./simplewallet
Minidump path:
^C
root@poo:/opt/qwertycoin# ./simplewallet --help
Minidump path: root@pool:/opt/qwertycoin#
root@poo:/opt/qwertycoin#
```

**Comment by speqtr** [comment]
This is a story about one broken line of debug output that silently crashed command line tools.

The root of all evil:
```
std::wcout << L"Minidump path: "; // << minidumpPath << ", id: " << minidumpId << std::endl;
```

@nnian sorry for that. Don't know how I could forget to remove it.

**Comment by speqtr** [comment]
Fixed in PR #128.

**Comment by nnian** [comment]
> 
> 
> This is a story about one broken line of debug output that silently crashed command line tools.
> 
> The root of all evil in `Breakpad.cpp`:
> 
> ```
> std::wcout << L"Minidump path: "; // << minidumpPath << ", id: " << minidumpId << std::endl;
> ```
> 
> @nnian sorry for that. Don't know how I could forget to remove it.

no problem. Everyone forgot to remove some debug outputs 💃
